/*
 * This file was automatically generated by EvoSuite
 * Sun May 26 21:10:03 GMT 2024
 */

package com.marketplace.vintage.exceptions;

import org.junit.Test;
import static org.junit.Assert.*;
import com.marketplace.vintage.exceptions.EntityAlreadyExistsException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class EntityAlreadyExistsException_ESTest extends EntityAlreadyExistsException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      EntityAlreadyExistsException entityAlreadyExistsException0 = new EntityAlreadyExistsException("com.marketplace.vintage.exceptions.EntityAlreadyExistsException");
  }
}
